rosdep/python.yaml

```diff
@@ -6576,6 +6576,10 @@ python3-ignition-math6:
   ubuntu:
     focal: [python3-ignition-math6]
     jammy: [python3-ignition-math6]
+python3-ikpy-pip:
+  '*':
+    pip:
```

Comment:
Standard pip disclaimer: ROS packages that depend on pip keys cannot be released into a ROS distribution. They can only be depended on by from-source builds. Because of this, system packages are highly preferred to pip packages.

Reply:
yes, fully aware... 🤷‍♂️

Reply:
`ikpy` is unfortunately not packaged upstream.